src/main/java/hyperledger/besu/java/client/ethconnecter/controller/TransactionController.java

```diff
@@ -1,54 +1,82 @@
 package hyperledger.besu.java.client.ethconnecter.controller;
```

Comment:
@nidhi-singh02 could you please work with @miir2709
1. To get the common response structure in place for all APIs.
2. To setup a global exception handler for all exception handling.